File: src/main/com/java/example/UserService.java
package main.com.java.example;

import java.sql.ResultSet;
import java.sql.SQLException;
import java.util.ArrayList;
import java.util.List;

public class UserService {

    public void createUsers(List<String> names) {
        for (String name : names) {
            DbHelper db = new DbHelper();
            db.update("INSERT INTO users (name) VALUES ('" + name + "')");
        }
    }

    public List<String> fetchUsers() {
        List<String> users = new ArrayList<>();
        DbHelper db = new DbHelper();
        ResultSet rs = db.query("SELECT name FROM users");
        try {
            while (rs != null && rs.next()) {
                users.add(rs.getString("name"));
            }
        } catch (SQLException e) {
        }
        return users;
    }
}

File: user_service.py
from db_helper import DbHelper

class UserService:
    def create_users(self, names):
        for name in names:
            db = DbHelper()
            db.update(f"INSERT INTO users (name) VALUES ('{name}')")
            # Connection creation in loop, SQL injection possible

    def fetch_users(self):
        db = DbHelper()
        result = db.query("SELECT name FROM users")
        users = []
        for row in result:
            users.append(row[0])
        return users
